databricks-iris/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/pipeline.py

```diff
@@ -25,12 +25,12 @@ def create_pipeline(**kwargs) -> Pipeline:
             node(
                 func=make_predictions,
                 inputs=["X_train@pandas", "X_test@pandas", "y_train@pandas"],
-                outputs="y_pred",
+                outputs="y_pred@pandas",
                 name="make_predictions",
             ),
             node(
                 func=report_accuracy,
-                inputs=["y_pred", "y_test@pandas"],
+                inputs=["y_pred@pandas", "y_test@pandas"],
                 outputs=None,
```

Comment:
I don't understand this change, can you elaborate a bit more?

Reply:
You cannot use `MemoryDataset` when using the `--node_names` parameter. Therefore, we have to specify a path for the prediction as well.